lib/App/cpm/CLI.pm

```diff
@@ -222,6 +222,25 @@ sub cmd_version {
     if ($App::cpm::GIT_DESCRIBE) {
         print "This is a self-contained version, $App::cpm::GIT_DESCRIBE ($App::cpm::GIT_URL)\n";
     }
+
+    print "perl version $^V ($^X)\n\n";
```

Comment:
Can you replace `$^V` by `$]`?
On perl 5.8, `$^V` is a v-string so that it is not printable.

Reply:
Oops. I think 7f60596b233554c0c6265be387ec4c6274c34ff9 is also okay.